x-either/x-either.cabal

```diff
@@ -13,7 +13,7 @@ description:           x-either.
 library
   build-depends:
                           base                        >= 4.6        && < 5
-                        , bifunctors                  >= 4.2        && < 5
+                        , bifunctors                  >= 4.2        && <= 5
```

Comment:
This constraint doesn't make sense, and P is almost certainly wrong. As far as I know we need `< 5` everywhere, but if we wanted it to be inclusive of 5.* the constraint would need to be `< 6` or `< 5.1` not `<= 5`.